indico/modules/events/registration/forms.py

```diff
@@ -158,3 +158,23 @@ def validate_users_field(self, field):
         if existing:
             raise ValidationError(_("There are already registrations with the following email addresses: {emails}")
                                   .format(emails=', '.join(sorted(existing))))
+
+
+class EmailRegistrantsForm(IndicoForm):
+    from_address = SelectField(_("From"), [DataRequired()])
+    cc_addresses = EmailField(_("CC"),
+                              description=_("Beware, addresses in this field will receive one mail per registrant."))
+    subject = StringField(_("Subject"), [DataRequired()])
+    body = TextAreaField(_("Email body"), [DataRequired()], widget=CKEditorWidget(simple=True))
+
+    def __init__(self, *args, **kwargs):
+        super(EmailRegistrantsForm, self).__init__(*args, **kwargs)
+        from_addresses = ['{} <{}>'.format(session.user.full_name, email)
+                          for email in sorted(session.user.all_emails, key=lambda x: x != session.user.email)]
+        self.from_address.choices = zip(from_addresses, from_addresses)
+        self.body.description = render_placeholder_info('registration-email', registration=None)
+
+    def validate_email_body(self, field):
```

Comment:
OK, I'll rename the method. I think the name `body` is better than `email_body`. We have `from_address`, `subject`, ... not `email_subject`, etc.